packages/number-input/src/Input/Input.styles.ts

```diff
@@ -30,6 +30,7 @@ export const inputBaseStyles = css`
   box-sizing: border-box;
   transition: ${transitionDuration.default}ms ease-in-out;
   transition-property: background-color;
+  padding: 0 0 0 ${spacing[1] * 3}px;
```

Comment:
Why `x3`? I feel like for numbers like this we should just use the actual number

Reply:
If we want to keep using `spacing` here I'd prefer adding instead of multiplying `spacing[2] + spacing[1]` (4 + 2)

Reply:
i'll use numbers, i think even with adding we'll run into similar issues